src/card.css.ts

```diff
@@ -9,7 +9,12 @@ export const actions = style([
     {marginTop: 16},
 ]);
 
-export const boxed = sprinkles({display: 'flex'});
+export const boxed = style([
+    sprinkles({display: 'flex'}),
+    {
+        isolation: 'isolate', // Needed to preserve border-radius with Video component and Safari
```

Comment:
this create a new stacking context, similar to the old hack of `translateZ(0)`

Reply:
oh, and Safari support this? from which version?

Reply:
since safari 8 :O